src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-extension.js

```diff
@@ -94,7 +94,8 @@
                             callback(data);
                         }.bind(this))
                         .fail(function(data) {
-                            app.sandbox.logger.log('data could not be loaded:', data);
+                            app.logger.log('data could not be loaded:', data);
+                            callback(null);
```

Comment:
Have to make sure that this is not a BC break. If the previous behavior was really that the application was breaking it should not be such a big deal.